docs/content/direct/binding.md

```diff
@@ -1,5 +1,179 @@
 # Binding workload with WEC
 
-This document is about BindingPolicy and Binding objects and the concept of downsync.
+This document is about associating WECs with workload objects. The
+primary concept is sometimes called "downsync", which confusingly
+refers to both the propagation and [transformation](transforming.md)
+of desired state from core to WECs _and_ the propagation and
+summarization of reported state from WECs to core.
 
-TODO: write it.
+## Binding Basics
+
+The user controls downsync primarily through API objects of kinds
+`BindingPolicy` and `Binding`. These go in a WDS and associate
+workload objects in that WDS with WECs, along with adding some
+modulations on how downsync is done.
+
+`BindingPolicy` is a higher level concept than `Binding`. KubeStellar
+has a controller that translates each `BindingPolicy` to a
+`Binding`. A user _could_ eschew the `BindingPolicy` and directly
+maintain a `Binding` object or let a different controller maintain the
+`Binding` object (TODO: check that this is true). The `Binding` object
```

Comment:
Unless we recommend doing that, or see a value in it, I don't see why we should add the " A user _could_ eschew the `BindingPolicy` and directly maintain a `Binding` object or let a different controller maintain the `Binding` object (TODO: check that this is true)."

Reply:
This was among the motivations given in the discussion about introducing the Binding object. It also is a bit of alignment with other projects, which is also a bit of value. So I definitely think it is worth mentioning.

Reply:
If  you believe it's valuable I'm OK with it.